src/console_reporter.cc

```diff
@@ -98,6 +98,20 @@ static void IgnoreColorPrint(std::ostream& out, LogColor, const char* fmt,
   va_end(args);
 }
 
+static std::string FormatTime(double time) {
+  // Align decimal places...
+  if (time < 1) {
+    return FormatString("%14.3f", time);
+  }
+  if (time < 10) {
+    return FormatString("%13.2f ", time);
+  }
+  if (time < 100) {
+    return FormatString("%12.1f  ", time);
+  }
+  return FormatString("%10.0f    ", time);
```

Comment:
Have you considered letting the format string figure out the necessary padding at least?
(see `man 3 vsprintf`)

Reply:
Could you clarify a bit?

The reason for the current implementation of `FormatString` is to ensure columns with the same place value align. For example.
```
Benchmark                                                Time                CPU         Iterations
----------------------------------------------------------------------------------------------------
BM_empty                                                0.572 ns            0.572 ns     1000000000
BM_empty/threads:72                                     0.021 ns            1.50  ns      958819032
BM_spin_empty/8                                         7.47  ns            7.47  ns      180868971
BM_spin_empty/512                                     954     ns          953     ns        1466617
BM_spin_empty/8192                                  15449     ns        15447     ns          90954
BM_spin_empty/8/threads:72                              0.258 ns           18.2   ns       72560520
BM_spin_empty/512/threads:72                           22.2   ns         1564     ns         872568
BM_spin_empty/8192/threads:72                         382     ns        26233     ns          38664
```

I'm not too familiar with printf specifiers, but if there is a tool I can use to do that, please let me know.

Reply:
Yeah, I don't love the extra width either. But being able to see sub-nanosecond results is worth the hit I think.